app/scripts/modules/core/src/presentation/SanitizedContent.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import DOMPurify from 'dompurify';
+
+export const SanitizedContent = ({
+  content,
+  inline,
+  config,
+}: {
+  content: string;
+  inline?: boolean;
+  config?: IDOMPurifyConfig;
+}) => {
+  const __html = DOMPurify.sanitize(content, config);
+  if (inline) {
+    return <span dangerouslySetInnerHTML={{ __html }} />;
+  }
+  return <div dangerouslySetInnerHTML={{ __html }} />;
+};
+
+export const sanitizeContent = (content: string, config = {}, inline = false) => (
```

Comment:
This is something I'm planning to use in a different PR (refactoring confirmation modal service to use React instead of Angular) but I am not totally sure I need it. If I don't, I'll remove it.